src/System.Runtime.Extensions/tests/System/Net/WebUtility.cs

```diff
@@ -296,37 +314,27 @@ public static void UrlDecodeToBytes(byte[] value, int offset, int count, byte[]
         }
 
         [Fact]
-        public static void UrlDecodeToBytes_Invalid()
+        public static void UrlDecodeToBytes_NullBytes_ThrowsArgumentNullException()
         {
-            Assert.Throws<ArgumentNullException>("bytes", () => WebUtility.UrlDecodeToBytes(null, 0, 1)); // Bytes is null
-
-            Assert.Throws<ArgumentOutOfRangeException>("offset", () => WebUtility.UrlDecodeToBytes(new byte[1], -1, 1)); // Offset < 0
-            Assert.Throws<ArgumentOutOfRangeException>("offset", () => WebUtility.UrlDecodeToBytes(new byte[1], 2, 1)); // Offset > bytes.Length
-
-            Assert.Throws<ArgumentOutOfRangeException>("count", () => WebUtility.UrlDecodeToBytes(new byte[1], 0, -1)); // Count < 0
-            Assert.Throws<ArgumentOutOfRangeException>("count", () => WebUtility.UrlDecodeToBytes(new byte[1], 0, 3)); // Count > bytes.Length
+            Assert.Throws<ArgumentNullException>("bytes", () => WebUtility.UrlDecodeToBytes(null, 0, 1));
         }
 
         [Theory]
-        [InlineData("a", 0, 1)]
-        [InlineData("a", 1, 0)]
-        [InlineData("abc", 0, 3)]
-        [InlineData("abc", 1, 2)]
-        [InlineData("abc", 1, 1)]
-        [InlineData("abcd", 1, 2)]
-        [InlineData("abcd", 2, 2)]
-        public static void UrlEncodeToBytes_NothingToExpand_OutputMatchesSubInput(string inputString, int offset, int count)
```

Comment:
I forgot to leave a comment, sorry. I directly added them as byte arrays here, avoiding the encoding step which is unecessary https://github.com/dotnet/corefx/pull/12123/files/23ce241ab72d3a276f7bc9c40b0be5eaf6b0688b#diff-a9595edda96423a24d85073fdb30a6f4R349